.circleci/config.yml

```diff
@@ -25,6 +26,9 @@ jobs:
           name: Check build images
           command: |
             docker images "fundocker/openshift*"
+            for service in $(ls project/docker/images); do
```

Comment:
Are you sure it's `project/docker/images` and not `docker/images`?

Reply:
I think after the checkout it `cd project`. Take a look at other for loops.

Reply:
Ok you're right. I was confused because when I sshed it was at the root...
I added some logging so we can follow and be sure that it tested all services.